<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatPage</class>
 <widget class="QWidget" name="ChatPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-tgInterface-Qt6-Debug/logoTogether.svg</normaloff>../build-tgInterface-Qt6-Debug/logoTogether.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #262626;
color: #FFFFFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>821</width>
     <height>441</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="widgetGroupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>51</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent; border: 0px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="addFriendQPushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/ChatPage/images/chatpage/addFriend.svg</normaloff>:/ChatPage/images/chatpage/addFriend.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="chatQPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/ChatPage/images/chatpage/completChatBule.svg</normaloff>:/ChatPage/images/chatpage/completChatBule.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="settingQPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/ChatPage/images/chatpage/settingWile.svg</normaloff>:/ChatPage/images/chatpage/settingWile.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="personQpushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/ChatPage/images/chatpage/my.svg</normaloff>:/ChatPage/images/chatpage/my.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="bellQpushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/ChatPage/images/chatpage/bell.svg</normaloff>:/ChatPage/images/chatpage/bell.svg</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>131</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="sendMessageGroupBox_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>341</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="sendMessageTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1E1E1E;border: 0px;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Type something...  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="sendMessageQpushButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>25</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/ChatPage/images/chatpage/send.svg</normaloff>:/ChatPage/images/chatpage/send.svg</iconset>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QPushButton" name="smileQpushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>25</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/ChatPage/images/chatpage/smile.svg</normaloff>:/ChatPage/images/chatpage/smile.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="addFileQpushButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>25</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/ChatPage/images/chatpage/addFile.svg</normaloff>:/ChatPage/images/chatpage/addFile.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="vocalMessageQpushButton">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>25</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/ChatPage/images/chatpage/vocalMessage.svg</normaloff>:/ChatPage/images/chatpage/vocalMessage.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="verticalSeparatorQpushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>25</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../rcs/images/chatpage/verticalSeparator.svg</normaloff>../rcs/images/chatpage/verticalSeparator.svg</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="recentMessageGroupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>171</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="buttonGroupBox">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>10</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1E1E1E;
border-radius: 8px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="goToChatQpushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #262626;
border-radius: 6px;</string>
      </property>
      <property name="text">
       <string>Chat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goToGroupQPushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Groups</string>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="friendsListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Friend</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="receiveMessageGroupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>341</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1E1E1E;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="chatQtextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="topWidgetGroupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>681</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="settingDoteQPushButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: 0px ;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/ChatPage/images/chatpage/settingDote.svg</normaloff>:/ChatPage/images/chatpage/settingDote.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="videoQPushButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: 0px ;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/ChatPage/images/chatpage/camera.svg</normaloff>:/ChatPage/images/chatpage/camera.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="callQPushButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: 0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rcs/qtrcs.qrc">
       <normaloff>:/ChatPage/images/chatpage/phone.svg</normaloff>:/ChatPage/images/chatpage/phone.svg</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="recentMessagelabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Recent Messages</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>56</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Acivity</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>1920</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>460</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../rcs/qtrcs.qrc"/>
 </resources>
 <connections/>
</ui>
